packages/core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -315,6 +315,7 @@ export default class Tooltip extends PureComponent<TooltipProps> {
     const content = this.renderTooltipContent;
     const dialogProps = {
       ...this.props,
+      position: position as DialogPosition,
```

Comment:
why was it added?

Reply:
Since need to pass the position as DialogPosition and not as TooltipPositionsType